roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2

```diff
@@ -141,10 +141,8 @@ router bgp {{ MD_Extended.vxlan.global.bgp_asn }}
       {% if peer.address_family_ipv4_unicast.weight %}
         weight {{ peer.address_family_ipv4_unicast.weight }}
       {% endif %}
-      {% if peer.address_family_ipv4_unicast.allow_as_in_number | default(defaults.vxlan.overlay_extensions.vrf_lites.switches.bgp_peers.address_family_ipv4_unicast.allow_as_in_number) %}
-        allowas-in {{ peer.address_family_ipv4_unicast.allow_as_in_number | default(defaults.vxlan.overlay_extensions.vrf_lites.switches.bgp_peers.address_family_ipv4_unicast.allow_as_in_number) }}
-      {% elif peer.address_family_ipv4_unicast.allow_as_in | default(defaults.vxlan.overlay_extensions.vrf_lites.switches.bgp_peers.address_family_ipv4_unicast.allow_as_in) %}
-        allowas-in
+      {% if peer.address_family_ipv4_unicast.allow_as_in_number %}
+        allowas-in {{ peer.address_family_ipv4_unicast.allow_as_in_number}}
```

Comment:
This would still setup "allow_as_in <value>" even if "allow_as_in" is set to 'false'. Please see the scenario as below:
```
allow_as_in: false
allow_as_in_number: 3
```
I think we should include another check with 'and' condition in the if statement to execute only when "allow_as_in" is also set to true (check the default as well).